modules/rest-root/src/main/java/org/elasticsearch/rest/root/MainResponse.java

```diff
@@ -97,6 +109,26 @@ public void writeTo(StreamOutput out) throws IOException {
         Build.writeBuild(build, out);
     }
 
+    private String getLuceneVersion() {
+        if (indexVersion != null) {
+            return indexVersion.luceneVersion().toString();
+        } else if (version.before(Version.V_8_10_0)) {
+            return IndexVersion.fromId(version.id).luceneVersion().toString();
+        } else {
+            return "unknown";
+        }
+    }
+
+    private String getMinimumWireCompatibilityVersion() {
+        // this is no longer computed from Version
+        return TransportVersion.current().equals(transportVersion) ? TransportVersion.MINIMUM_COMPATIBLE.toString() : "unknown";
```

Comment:
This seems to be an unrelated change to adding index version here. I think it should be raised separately (same for min index version). These should probably be passed along as arguments to the response, instead of looked up statically?